blockchain/ethereum.go

```diff
@@ -28,25 +28,22 @@ import (
 	"github.com/smartcontractkit/chainlink-testing-framework/utils"
 )
 
+var debugID int
```

Comment:
It deserves some comments, what kind of `debugID` is this, or it's temporary?

Reply:
Extraneous, removed